_layouts/default.html

```diff
@@ -1,38 +1,63 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="{{ site.lang | default: "cs-CZ" }}">
+
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="chrome=1">
-    <meta property="og:image" content="http://sijemerousky.cz/images/rouska2.png">
```

Comment:
@BecAlenvers Don't remove this, please. It is used for preview while sharing links on social media.